.github/workflows/tests.yaml

```diff
@@ -34,21 +34,38 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
 
       - name: Load secrets
-        working-directory: tests/functional_tests
+        working-directory: tests/functional_tests/tools/virtualenv/keys
         run: |
           mkdir -p ~/.atsign/keys/
-          echo "${{ secrets.ATKEYS_12ALPACA }}" > ~/.atsign/keys/@12alpaca_key.atKeys
-          echo "${{ secrets.ATKEYS_12SNOWBOATING }}" > ~/.atsign/keys/@12snowboating_key.atKeys
+          ls -la
```

Comment:
Looks like a stray debug line

Reply:
Are you saying the debug lines should be removed?

Reply:
Done in https://github.com/atsign-foundation/at_c/pull/476/commits/e91dc26f51097b190c93ec3f815b6f421e0fe5a6